lib/calc-dep-flags.js

```diff
@@ -11,7 +11,7 @@ const calcDepFlags = (tree, resetRoot = true) => {
     tree,
     visit: node => calcDepFlagsStep(node),
     filter: node => node,
-    getChildren: node => [...node.edgesOut.values()].map(edge => edge.to),
+    getChildren: (node, tree) => [...tree.edgesOut.values()].map(edge => edge.to),
```

Comment:
This is the optimally unconfusing way to do it, but it tickles me that the whole diff could've been:

```
- getChildren: node => ....
+ getChildren: (_,node) => ....
```